### PR TITLE
Render Lists of Dragons using useSelector() and React Bootstrap

### LINKED ISSUE #14: Team3 [4pt] Display dragons - Lists render
- Use `useSelector()` Redux Hook to select the state slices and render lists of dragons in corresponding routes. i.e.:
```javascript
// get rockets data from the store
const rockets = useSelector(state => state.rockets);
```

- You can style the whole application "by hand" or you could use [React Bootstrap](https://react-bootstrap.github.io/), a UI library that could speed up the process. This is a popular library and working with its components would be good practice.

- Render a list of dragons (as per design). For the image of a dragon use the first image in the array of `flickr_images`.

### PR DESCRIPTION
**Pull request by PabloBona**
**Description:**

- Implemented `useSelector()` Redux Hook to select the state slices containing dragons' data in corresponding routes.
- Fetched the rockets data from the store using `useSelector()` and stored it in the `rockets` variable.
- Styled the whole application using React Bootstrap, a popular UI library that provides pre-built components for faster development.
- Rendered a list of dragons as per the design, utilizing the first image in the array of flickr_images as the image of each dragon.
- Ensured that the rendered lists of dragons display the required information in the desired format.

### CODE REVIEW

File: src/components/DragonCard.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+const DragonCard = ({ dragon }) => (
+  <div className="dragon-card rounded my-3" key={dragon.id}>
+    <div className="row">
+      <div className="col-sm-12 col-md-5">
+        {dragon.name === 'Dragon 1' ? (
+          <img className="w-100" src={dragon.flickr_images[0]} alt={dragon.name} />
+        ) : (
+          <img className="w-100" src={dragon.flickr_images[0]} alt={dragon.name} />
+        )}
```
Comment (hajnaloltyan) [inline src/components/DragonCard.jsx]: Hi, @PabloBona this looks the same for me, in both thruthy and falsy condition, please consider updating it, it's a minor issue.

Comment (PabloBona) [reply src/components/DragonCard.jsx]: Yes you're right; the images have different sizes and at the beginning I wanted to show the dragon 1 the img 2 and vice versa.
But in the end one of the requirements was to show only image 0. And never change it!

Comment (PabloBona) [reply src/components/DragonCard.jsx]: @hajnaloltyan 
Changes Made It !

**Review by Alibaba2023 — APPROVED** [review]
# Hi @PabloBona, 
congratulations, your project is complete.

**Review by hajnaloltyan — APPROVED** [review]
Hi, @PabloBona, congratulations your task is complete, it's time to merge it. 👏👏👏

**Pull request merged**